app/src/main/java/ingsw/group1/findmyphone/contacts/SMSContactManager.java

```diff
@@ -33,14 +31,28 @@
      *
      * @param applicationContext {@link Context} of the application
      */
-    public SMSContactManager(Context applicationContext) {
+    private SMSContactManager(Context applicationContext) {
         contactDatabase = Room.databaseBuilder(applicationContext, SMSContactDatabase.class,
                 CONTACTS_DB_NAME)
                 .enableMultiInstanceInvalidation()
                 .allowMainThreadQueries()
                 .build();
     }
 
+    /**
```

Comment:
I did the Singleton too. I marked it on trello an hour ago. It is in the related contact branch.
Tomorrow I will open a pull request with the branch `feature/contacts`.

Reply:
That's strange, I saw it free and before doing anything I marked it done xD